lib/widgets/story/activity/jumble_words.dart

```diff
@@ -0,0 +1,126 @@
+import 'dart:math';
+
+import 'package:built_collection/built_collection.dart';
+import 'package:flutter/material.dart';
+import 'package:jamaica/games/jumbled_words_game.dart';
+import 'package:jamaica/widgets/bento_box.dart';
+import 'package:jamaica/widgets/cute_button.dart';
+
+TextStyle textStyle({double fSize = 25, color: Colors.red}) =>
+    TextStyle(fontSize: fSize, fontFamily: 'Bungee', color: color, shadows: [
+      Shadow(
+        blurRadius: 3.0,
+        color: Colors.black,
+      )
+    ]);
+
+class _ChoiceDetail {
+  String choice;
+  int index;
+  bool appear;
+
+  _ChoiceDetail({this.choice, this.appear = true, this.index});
+  @override
+  String toString() =>
+      '_ChoiceDetail(choice: $choice, appear: $appear,index: $index, )';
+}
+
+class JumbleWords extends StatefulWidget {
```

Comment:
this should be a game (like the other ones). you have to implement onGameOver. Check with Kiran/Nikhil how.